src/content/releases/_index.en.md

```diff
@@ -5,6 +5,29 @@ pre = "<b>4. </b>"
 weight = 15
 +++
 
+## v0.2.0 Overlapping CIDR support
+
+> This release is focused on overlapping CIDR support between clusters
+
+* Support for Overlapping CIDRs between clusters (globalnet)
+* Enhanced e2e scripts, which will be shared between repositories in the shipyard project (ongoing work)
+* Local registry in make e2e, improves e2e deployment.
+* Refactoring for support pluggable drivers (in preparation for [WireGuard](https://www.wireguard.com/) support)
+
```

Comment:
Should we mention "Added a documentation website"?

Reply:
Nah, they can see if they are on the website already, it's more like a co-lateral :)